tests/unit/amo/components/TestCollection.js

```diff
@@ -1268,6 +1268,66 @@ describe(__filename, () => {
         username: detail.author.username,
       }),
     );
+
+    sinon.assert.notCalled(root.instance().props.history.push);
+  });
+
+  it('updates the page when removeAddon removes the last addon on the page', () => {
+    const history = createFakeHistory();
```

Comment:
You should pass this to `renderComponent()` to make the test more robust. In other words, you want to keep it in sync with your assertion.